### ISSUE TITLE
RoboCup2024 - Comments and suggestions

### ISSUE THREAD

**Issue by ARTenshi** [RoboCupAtHome/RuleBook#882]
## Is your idea/suggestion related to a problem? Please describe.
The RoboCup 2024 is over and it's time to review the competition while the experience is fresh. 

## Additional context
Was the organization as expected? Are the rules clear? Does the rulebook reflect your idea of a service robot? Please discuss it with your team and share your comments here.

**Comment by LoyVanBeek** [comment]
I was just a visitor with my kid this year, mostly during the finals and I found things very hard to follow. And I say this as a former TC...

IIRC there were some TV-screens around, which could be used to make clear what is going on. Years ago I prepared something for that https://github.com/RoboCupAtHome/vizbox, but even my 'own' team is not using it. And that makes sense, since all of those things are additional things that need to be set up and don't yield anything for the team itself. 

Even just having a static presentation slide on those screens of what the robot's task is would be nice already.

**Comment by PetervDooren** [comment]
> I was just a visitor with my kid this year, mostly during the finals and I found things very hard to follow. And I say this as a former TC...
> 
> IIRC there were some TV-screens around, which could be used to make clear what is going on. Years ago I prepared something for that https://github.com/RoboCupAtHome/vizbox, but even my 'own' team is not using it. And that makes sense, since all of those things are additional things that need to be set up and don't yield anything for the team itself.
> 
> Even just having a static presentation slide on those screens of what the robot's task is would be nice already.

A few weeks before the competition I checked out the vizbox and tried to get it to work. Unfortunately it seems to no longer work with the current python version. I do think our competition would benefit from using it. It would require some work to get the vizbox up and running. 

But I think a bigger issue is the audience not being able to hear the robots at all and vice versa. No speaker on the robot will be able to compete with a conference hall. The only solution I can imagine is to attach a sound receiver to the robot which is hooked up to speakers next to the arenas. But this would require having a standard interface, maybe just an audio jack. I heard this was tried before but gave problems with interference with remote e-buttons.

Another problem is that the walls of the arena often prevent the audience from seeing what is going on. I would suggest to drastically lower the front wall of the arena to a height of 50cm so the audience can see in. We can still use retractable barriers or likewise to prevent people from leaning or stepping in. The wall would still be high enough for the robots to do localization with them using their Lidars. A result of this is that the audience can see what is happening from the audience stands. Which prevents a crowd from forming very close to the arena itself.

**Comment by MrKeith99** [comment]
> Unfortunately it seems to no longer work with the current python version. I do think our competition would benefit from using it. It would require some work to get the vizbox up and running.

About this, our team (SOBITS) actually managed to update it to Python3 version, for the last year's competition.
Here is our fork if you want to have a look at it: https://github.com/TeamSOBITS/vizbox

However, I felt that even if the teams get ready with VizBox, there was no structure that enabled the visualization of it during the competition. (In the RulesBook for Bordeux2023, there was a section in which the team were required to make use of that tool.)

> Another problem is that the walls of the arena often prevent the audience from seeing what is going on. 

I agree that additional setup in the field might be required. However this is not the only problem. I think that the required network to visualize VizBox over the screen might require of a wireless connection robot-external computer, which is not allowed? (I am not quite sure about this). Some kind of exceptions in the rules might be required.

**Comment by MrKeith99** [comment]
I have just check that in this year's RulesBook, it is still there.
![image](https://github.com/user-attachments/assets/caa03c03-9fb8-46c7-b48d-a13b9d92d3d7)

**Comment by MatthijsBurgh** [comment]
This would enforce teams to use the arena network. We have been running our robot without any network connection. This can open a whole can of very messy stuff
